src/roman_datamodels/datamodels.py

```diff
@@ -300,6 +303,57 @@ class GuidewindowModel(DataModel):
     pass
 
 
+class ModelContainerModel(DataModel, Sequence):
+    # Needs a number of methods to properly handle the contents
+    def __init__(self, init=None, asn_schema= None, asn_file_path=None, model_file_path=None, iscopy=False, **kwargs):
```

Comment:
It looks like there's a mixture of parameters - those used to initialize the model from association table and also from some other input. Add a documentation section to clarify what the parameters are.